src/components/common/Forms/MapAddressForm/MapAddressForm.tsx

```diff
@@ -0,0 +1,329 @@
+          <div className={styles.nickname}>
+            <div className={styles.inputHint}>Nickname</div>
+            <Input
+              placeholder="My house"
+              value={this.state.values.nickname}
+              error={this.state.errors.nickname}
+              onChange={(val: string) => {
+                this.updateFieldForm("nickname", val);
+              }}
+            />
+          </div>
+
+          <div className={styles.firstName}>
+            <div className={styles.inputHint}>First Name</div>
+            <Input
+              placeholder="Jane"
+              value={this.state.values.firstName}
+              error={this.state.errors.firstName}
+              onChange={(val: string) => {
+                this.updateFieldForm("firstName", val);
+              }}
+            />
+          </div>
+
+          <div className={styles.lastName}>
+            <div className={styles.inputHint}>Last Name</div>
+            <Input
+              placeholder="Doe"
+              value={this.state.values.lastName}
+              error={this.state.errors.lastName}
+              onChange={(val: string) => {
+                this.updateFieldForm("lastName", val);
+              }}
+            />
+          </div>
+
+          <div className={styles.address1}>
+            <div className={styles.inputHint}>Address Line 1</div>
+            <Input
+              placeholder="123 Street Name"
+              value={this.state.values.addressLine1}
+              error={this.state.errors.addressLine1}
+              onChange={(val: string) => {
+                this.updateFieldForm("addressLine1", val);
+              }}
+            />
+          </div>
+
+          <div className={styles.address2}>
+            <div className={styles.inputHint}>Address Line 2</div>
+            <Input
+              placeholder="Apt. #22A"
+              value={this.state.values.addressLine2}
+              error={this.state.errors.addressLine2}
+              onChange={(val: string) => {
+                this.updateFieldForm("addressLine2", val);
+              }}
+            />
+          </div>
+
+          <div className={styles.city}>
+            <div className={styles.inputHint}>City</div>
+            <Input
+              placeholder="City"
+              value={this.state.values.city}
+              error={this.state.errors.city}
+              onChange={(val: string) => {
+                this.updateFieldForm("city", val);
+              }}
+            />
+          </div>
+
+          <div className={styles.state}>
+            <div className={styles.inputHint}>State</div>
+            <StyledSelect
+              options={usaStates}
+              values={[usaStates[0]]}
+              labelField="abbreviation"
+              valueField="abbreviation"
+              dropdownPosition="auto"
+              className={styles.dropDownStyle}
+              onChange={(value) => this.updateFieldForm("state", value)}
```

Comment:
This sometimes causes a validation error as the values.state field gets set with an object instead of the state string.
```
"state must be a `string` type, but the final value was: `[
  {
    "name": "\"Connecticut\"",
    "abbreviation": "\"CT\""
  }
]`."
```